### ISSUE TITLE
TimberImage doesn't work in .twig files

### ISSUE THREAD

**Issue by joeldbirch** [jarednova/timber#842]
Any call to TimberImage from within a .twig file results in a fatal error. Have tried with hardcoded image ID, url, etc. For example:

```html
<img src="{{TimberImage(post.get_field('hero_image')).src}}" />
```
Results in this error:
```
[02-Feb-2016 13:30:07 UTC] PHP Fatal error:  Call to undefined method Twig_SimpleFunction::compile() in /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Node/Expression/Call.php on line 27
[02-Feb-2016 13:30:07 UTC] PHP Stack trace:
[02-Feb-2016 13:30:07 UTC] PHP   1. {main}() /Users/htdocs/my-website/index.php:0
[02-Feb-2016 13:30:07 UTC] PHP   2. require() /Users/htdocs/my-website/index.php:17
[02-Feb-2016 13:30:07 UTC] PHP   3. require_once() /Users/htdocs/my-website/wp-blog-header.php:16
[02-Feb-2016 13:30:07 UTC] PHP   4. include() /Users/htdocs/my-website/wp-includes/template-loader.php:89
[02-Feb-2016 13:30:07 UTC] PHP   5. Timber::render() /Users/htdocs/my-website/wp-content/themes/my-theme/tmpl-home.php:11
[02-Feb-2016 13:30:07 UTC] PHP   6. Timber::fetch() /Users/htdocs/my-website/wp-content/plugins/timber-library/timber.php:383
[02-Feb-2016 13:30:07 UTC] PHP   7. Timber::compile() /Users/htdocs/my-website/wp-content/plugins/timber-library/timber.php:368
[02-Feb-2016 13:30:07 UTC] PHP   8. TimberLoader->render() /Users/htdocs/my-website/wp-content/plugins/timber-library/timber.php:329
[02-Feb-2016 13:30:07 UTC] PHP   9. Twig_Environment->render() /Users/htdocs/my-website/wp-content/plugins/timber-library/lib/timber-loader.php:69
[02-Feb-2016 13:30:07 UTC] PHP  10. Twig_Environment->loadTemplate() /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Environment.php:347
[02-Feb-2016 13:30:07 UTC] PHP  11. Twig_Environment->compileSource() /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Environment.php:396
[02-Feb-2016 13:30:07 UTC] PHP  12. Twig_Environment->compile() /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Environment.php:667
[02-Feb-2016 13:30:07 UTC] PHP  13. Twig_Compiler->compile() /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Environment.php:651
[02-Feb-2016 13:30:07 UTC] PHP  14. Twig_Node_Module->compile() /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Compiler.php:86
[02-Feb-2016 13:30:07 UTC] PHP  15. Twig_Node_Module->compileTemplate() /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Node/Module.php:52
[02-Feb-2016 13:30:07 UTC] PHP  16. Twig_Compiler->subcompile() /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Node/Module.php:82
[02-Feb-2016 13:30:07 UTC] PHP  17. Twig_Node->compile() /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Compiler.php:97
[02-Feb-2016 13:30:07 UTC] PHP  18. Twig_Node->compile() /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Node.php:110
[02-Feb-2016 13:30:07 UTC] PHP  19. Twig_Node_Block->compile() /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Node.php:110
[02-Feb-2016 13:30:07 UTC] PHP  20. Twig_Compiler->subcompile() /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Node/Block.php:34
[02-Feb-2016 13:30:07 UTC] PHP  21. Twig_Node->compile() /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Compiler.php:97
[02-Feb-2016 13:30:07 UTC] PHP  22. Twig_Node_Print->compile() /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Node.php:110
[02-Feb-2016 13:30:07 UTC] PHP  23. Twig_Compiler->subcompile() /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Node/Print.php:30
[02-Feb-2016 13:30:07 UTC] PHP  24. Twig_Node_Expression_Filter->compile() /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Compiler.php:97
[02-Feb-2016 13:30:07 UTC] PHP  25. Twig_Node_Expression_Call->compileCallable() /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Node/Expression/Filter.php:37
[02-Feb-2016 13:30:07 UTC] PHP  26. Twig_Node_Expression_Call->compileArguments() /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Node/Expression/Call.php:30
[02-Feb-2016 13:30:07 UTC] PHP  27. Twig_Compiler->subcompile() /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Node/Expression/Call.php:70
[02-Feb-2016 13:30:07 UTC] PHP  28. Twig_Node_Expression_Function->compile() /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Compiler.php:97
[02-Feb-2016 13:30:07 UTC] PHP  29. Twig_Node_Expression_Call->compileCallable() /Users/htdocs/my-website/wp-content/plugins/timber-library/vendor/twig/twig/lib/Twig/Node/Expression/Function.php:36
```

**Comment by jarednova** [comment]
@joeldbirch: the reported error in the log doesn't seem like it should be triggered by the `.twig` sample you provided. Can you try removing that line entirely and see if you still get the error? I have a feeling the problem might lie further up the chain

**Comment by joeldbirch** [comment]
Removing the line results in no error. That line is one of many permutations I tried, but any reference to TimberImage from inside a .twig file causes that fatal error. I'm stumped!

**Comment by connorjburton** [comment]
Hi @joeldbirch are you still experiencing this?

**Comment by joeldbirch** [comment]
I'll double check and get back to you. Thanks.

**Comment by joeldbirch** [comment]
You know what? It works now. ¯\_(ツ)_/¯

I have since updated PHP from 5.4.something to 5.5.9, but I'm not sure that was it really. Unfortunately I didn't commit the broken code to git, so for all that I spent hours staring at it there could still have been some silly mistake I missed, I suppose.

Anyway, I'm pretty excited to be up and running now. Sorry I couldn't provide more info to figure out if it was a real issue. And thanks @connorjburton for the nudge to recheck it!
